oxd-server/src/main/java/org/gluu/oxd/server/Utils.java

```diff
@@ -131,4 +134,19 @@ public static Properties loadPropertiesFromFile(String filename, Properties prop
             throw new RuntimeException("Could not load properties from file " + filename, e);
         }
     }
+
+    public static String getOxdVersion() {
+        try {
+            MavenXpp3Reader reader = new MavenXpp3Reader();
```

Comment:
Reader is not close. We dont want to leave unreleased resources.

Reply:
`FileReader` is Closeable.

MavenXpp3Reader just parse the xml file so doesn't have close method.

Reply:
Got it. `FileReader` within `try-with-resources` is fine.